apis/python/src/tiledbsoma/common_nd_array.py

```diff
@@ -38,8 +38,12 @@ def create(
         :param type: The Arrow type to be stored in the NDArray.
             If the type is unsupported, an error will be raised.
 
-        :param shape: the length of each dimension as a sequence, e.g., ``[100, 10]``.
-            All lengths must be in the positive int64 range.
+        :param shape: The maximum capacity of the dataframe, including room for any
+            intended future appends,as a sequence. E.g. ``[100, 10]``.  All
```

Comment:
please use a tuple, not a list, in the example as it is not intended to be a mutable arg

Reply:
oh my bad @bkmartinjr i have tried to be _very_ discipliined about using tuples not lists!!!! thank you!